robosat/features/core.py

```diff
@@ -102,38 +102,40 @@ def contours(mask):
       The detected contours as a list of points and the contour hierarchy.
 
     Note: the hierarchy can be used to re-construct polygons with holes as one entity.
-    '''
+    """
 
-    _, contours, hierarchy = cv2.findContours(mask, cv2.RETR_TREE, cv2.CHAIN_APPROX_SIMPLE)
+    _, contours, hierarchy = cv2.findContours(
+        mask, cv2.RETR_TREE, cv2.CHAIN_APPROX_SIMPLE
+    )
```

Comment:
Wow these parenthesis placements are ugly :see_no_evil: Is there a way to change it?

Reply:
This was also fixed with `--line-length 120`